Source/WebKit/UIProcess/API/glib/WebKitDownload.cpp

```diff
@@ -407,23 +411,26 @@ void webkitDownloadFinished(WebKitDownload* download)
 String webkitDownloadDecideDestinationWithSuggestedFilename(WebKitDownload* download, const CString& suggestedFilename, bool& allowOverwrite)
 {
     if (download->priv->isCancelled)
-        return emptyString();
+        return { };
```

Comment:
This must be fine since the tests are still passing, but I recently learned that the default String is the null string, which is not the same as the empty string, so I'm going to be a lot more careful about converting emptyString() to `{ }` in the future.

Reply:
When using empty string works, null string should work too, because isEmpty() returns true for null strings. It dosnt' work the other way around.